lib/benchee/benchmark/runner.ex

```diff
@@ -146,12 +138,15 @@ defmodule Benchee.Benchmark.Runner do
                       current_time: current_time, end_time: end_time
                     }) when current_time > end_time do
     # restore correct order - important for graphing
-    %Scenario{scenario | run_times: Enum.reverse(run_times)}
+    %Scenario{scenario | run_times: run_times |> List.flatten |> Enum.sort}
```

Comment:
Huh, can't do this! We don't want the run_times to be sorted but we want them to be in the order that they occurred (hence reversing them) so that formatters can plot them as time goes by. I see I gotta make the test for this more robust as sorting comes up with the same result :D

Reply:
ah, I just realized that of course if we have just big chunks recorded in parallel it will be out of order either way unless we go around and insert the first next to the first etc. which would still be cheating... might just wanna display a warning in the HTML formatter then :|

Reply:
Actually, I fixed this in #168. Instead of the cons operator, we can use `[] ++ []` to keep stuff in order. Then we don't even need to reverse it!

Reply:
but `++` also only works when we append a larger list to a smaller list (what we do in #168 as far as I can see), which would still need reversing imo. If we appended (which doesn't need reversing) it'd be super slow as we'd have to walk the linked list fully all the time :(